### ISSUE TITLE
autodl-irssi issue

### ISSUE THREAD

**Issue by Bensawsome** [binhex/arch-rtorrentvpn#100]
Hello,
So basically I am trying to setup autodl-irssi and for whatever reason it refuses to connect to the irc server when I am using a VPN. When I disabled the vpn it connected fine. Can I get some assistance with this?

**Comment by Turbo123T3** [comment]
![bitmoji](https://render.bitstrips.com/v2/cpanel/ef2ea898-c606-49aa-baeb-f882a9f55e21-9efc18f9-c48b-4472-99ac-a54a6d2846c5-v1.png?transparent=1&palette=1&width=246)

**Comment by disrupted** [comment]
I am also having issues connecting to one of the IRC channels. Do you happen to know where I can study the logs for autodl-irssi with this docker?

**Comment by binhex** [comment]
As I do not use irssi i need to have step-by-step instructions on how to replicate the issue the more details the better.

**Comment by Bensawsome** [comment]
Ok well I had time to look into this more and apparently it has nothing to do with irssi not working it's the VPN ip is banned from the irc network. If anyone else runs into this the way I checked why it wasn't able to connect was I went into the rtorrentvpn console and open irssi (by typing irssi into the terminal) and it showed me "Link closed, user has been banned. Proxy detected." The workaround I use is to have rtorrentvpn connect without a vpn and only have rtorrent SAVE the torrent files to a watch folder. Then install either delugevpn or qbittorrentvpn and have one of those look for new torrents in the same watch folder and make sure it is behind a vpn.
Thank you,
Bensawsome
